test/grpc-creds/generate.sh

```diff
@@ -21,5 +21,7 @@ for SERVICE in publisher nonce ra ca sa va rva ; do
   minica -domains "${SERVICE}.boulder,${SERVICE}1.boulder,${SERVICE}2.boulder"
 done
 
+minica -ip-addresses 10.77.77.77,10.88.88.88
```

Comment:
```suggestion
IP_ADDRS="10.77.77.77,10.88.88.88"
minica -ip-addresses "${IP_ADDRS}"
mv "${IP_ADDRS%%,*}" doh
```

Using bash shell string manipulation, we can access the word before the first comma.